### DISCUSSION TITLE
Cannot copy to clipboard, make sure you are accessing the page through https

### DISCUSSION THREAD

**Discussion by seriaati** [immich-app/immich#14273] [Q&A] [answered]
I learned from [this issue](https://github.com/immich-app/immich/discussions/14251) that I need to directly go to the ip address of the server to upload files larger than 100MB, but when I try to share my photos, immich says "Error: Cannot copy to clipboard, make sure you are accessing the page through https."

So this kinda confuses me because there is a limitation of sharing with only https but https has an upload limit of 100MB? My knowledge about https is only through cloudflare tunnels, I guess there are maybe ways to enable https without having an upload limit; if that is the case please teach me how to do so, thanks.

**Comment by alextran1502** [comment, marked as answer]
There are other ways to have `https` by using reverse-proxy with either Caddy or NGINX

https://immich.app/docs/administration/reverse-proxy/

**Comment by mmomjian** [comment]
To add to this, this is a web browser limitation that prevents clipboard access over HTTP, immich can’t fix it
